fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -47,6 +47,28 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     private static final Logger LOG = LogManager.getLogger(SessionVariable.class);
 
     public static final String EXEC_MEM_LIMIT = "exec_mem_limit";
+
+    /**
+     * configure the mem limit of load process on BE.
+     * Previously users used exec_mem_limit to set memory limits.
+     * To maintain compatibility, the default value of load_mem_limit is 0,
+     * which means that the load memory limit is still using exec_mem_limit.
+     * Users can set a value greater than zero to explicitly specify the load memory limit.
+     * This variable is mainly for INSERT operation, because INSERT operation has both query and load part.
+     * Using only the exec_mem_limit variable does not make a good distinction of memory limit between the two parts.
+     */
+    public static final String LOAD_MEM_LIMIT = "load_mem_limit";
+
+    /**
+     * The mem limit of query on BE. It takes effects only when enabling pipeline engine.
+     * - If `query_mem_limit` > 0, use it to limit the memory of a query.
+     *   The memory a query able to be used is just `query_mem_limit`.
+     * - Otherwise, use `exec_mem_limit` to limit the memory of a query.
```

Comment:
Maybe in future version, we should remove the limit if `query_mem_limit == 0` , but not use `exec_mem_list` as limit?

Reply:
Maybe for the compatibility, we can not remove `exec_mem_list` easily.